docker/nginx/default.conf

```
@@ -2,7 +2,12 @@ server {
     listen       80;
     server_name  localhost;
 
-    location /openzaak {
-        proxy_pass   http://web:8000;
+    location /private-media {
+        alias /private-media;
```

```suggestion
        internal;
        alias /private-media;
```

As discussed let's make it more secure